source/index.html

```diff
@@ -84,65 +81,55 @@ <h2 class="offer__title">
           </li>
           <li class="operation__item">
-            <p class="operation__number">2</p>
-            <p class="operation__description operation__description--powder">
-              Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
-            </p>
+            Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
           </li>
           <li class="operation__item">
-            <p class="operation__number">3</p>
-            <p class="operation__description operation__description--nutrition">
-              Замените один-два приема обычной еды на наше функциональное питание.
-            </p>
+            Замените один-два приема обычной еды на наше функциональное питание.
           </li>
           <li class="operation__item">
-            <p class="operation__number">4</p>
-            <p class="operation__description operation__description--changes">
-              Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
-            </p>
+            Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
           </li>
         </ol>
       </div>
     </section>
 
-    <section class="slider">
-      <h2 class="slider__title">
-        Живой пример
-      </h2>
-      <p class="slider__text">
-        Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без
-        изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
-      </p>
-      <div class="slider__statistic">
-        <dl class="slider__list">
-          <div class="slider__list-item">
-            <dt class="slider__description-statistics">5 КГ</dt>
-            <dd class="slider__description-text">снижение веса</dd>
-          </div>
-          <div class="slider__list-item">
-            <dt class="slider__description-statistics">60 ДНЕЙ</dt>
-            <dd class="slider__description-text">затрачено времени</dd>
-          </div>
-        </dl>
-        <p class="slider__description-result">Затраты на питание: 15 000 РУБ.</p>
-      </div>
+    <section class="example">
+      <div class="example__wrapper">
+        <h2 class="example__title">
+          Живой пример
+        </h2>
+        <p class="example__text">
+          Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy Slim. Отличный результат без
+          изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по 16 часов в день.
+        </p>
+        <div class="example__statistic">
+          <dl class="example__list">
+            <div class="example__list-item">
+              <dd class="example__description-statistics">5 кг</dd>
+              <dt class="example__description-text">снижение веса</dt>
+            </div>
+            <div class="example__list-item">
+              <dd class="example__description-statistics">60 ДНЕЙ</dd>
```

Comment:
Ну это не только к "кг" относился коммент, а вообще ко всему.

```suggestion
              <dd class="example__description-statistics">60 дней</dd>
```

Reply:
Исправила